codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,13 +175,19 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
+        FlowState state;
```

Comment:
@buchgr - Yip

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project netty-codec-http2: Compilation failure
[ERROR] .../codec/http2/DefaultHttp2RemoteFlowController.java:[181,12] error: variable state might not have been initialized